rmw_connext_cpp/CMakeLists.txt

```diff
@@ -60,11 +67,6 @@ if(WIN32)
             "RMW_CONNEXT_SHARED_BUILDING_DLL")
 endif()
 
-register_rmw_implementation(
-  "c:rosidl_typesupport_connext_c"
-  "cpp:rosidl_typesupport_connext_cpp"
-)
-
```

Comment:
it doesn't need to, I just did it to make it match rmw_connext_dynamic_cpp/CMakeLists.txt